tests/Unit/PublicPropertyManagerTest.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+namespace Tests\Unit;
+
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Support\Str;
+use Livewire\Exceptions\CannotRegisterPublicPropertyWithoutExtendingThePropertyHandlerException;
+use Livewire\LivewireProperty;
+use Livewire\Livewire;
+
+class PublicPropertyManagerTest extends TestCase
+{
+    public function setUp(): void
+    {
+        parent::setUp();
+
+        if (version_compare(PHP_VERSION, '7.4', '<')) {
+            $this->markTestSkipped('Skip public property tests if the version is below PHP 7.4');
+        }
+
+        require_once __DIR__.'/PublicPropertyManagerStubs.php';
```

Comment:
Maybe move the classes to the end of this test file, instead of including the stubs file here, as that keeps it consistent with Livewire's other tests.